extensions/amp-story/1.0/_locales/es.js

```diff
@@ -34,16 +34,16 @@ export default {
     string: 'Más sobre los resultados de AMP',
   },
   [LocalizedStringId.AMP_STORY_HINT_UI_NEXT_LABEL]: {
-    string: 'Toque Siguiente',
+    string: 'Toca Siguiente',
```

Comment:
Just curious as I'm not a spanish speaker: is this expected to be different in `es-419` and `es`?

Reply:
The formal/informal registry is slightly different. "Toque" could work in both, but "Toca" is probably better in es-ES.

Reply:
Sorry for the audacity, but I think it would be preferable to simply translate "Toque" in both cases, because "Toque Volver" does not fit on some devices (see capture of an iPhone 6/7/8 in zoom mode, which shares viewport size with iPhone 5SE, 320 x 460 px).

<img width="320" alt="image-1" src="https://user-images.githubusercontent.com/5907868/45995586-ef784a00-c098-11e8-8db5-147628d61479.png">

 The arrows above show the direction and you can save the word "Previous" or "Next". I think these messages are displayed when the user makes the "swipe" gesture, and we just want to say: "do not swipe, touch" and the "back" and "forward" are shown by arrows icons.

The word "Toque" has the advantage that it is a verb, but also a noun: and it can also be understood as "dar un toque aquí".



![imagen](https://user-images.githubusercontent.com/5907868/46012176-a3dd9480-c0c8-11e8-805d-925335df079d.png)


Alternatively, if you want to keep the two words, you would have to remove the absolute positioning of the element ".i-amphtml-story-hint-placeholder" in the file "/extensions/amp-story/1.0/amp-story-hint.css" since being positioned in an absolute way, it escapes the normal flow of blocks, and specifically the flex container of the left sidebar that measures a quarter of the width and that is where that text should "fit". 

And also can be deleted  this rule: "top: 50%" because its father flex block has alig-items center.

.i-amphtml-story-hint-placeholder {
    top: 50%!important; // can ve removed
    position: absolute; // can ve removed
(...)
}

![h1](https://user-images.githubusercontent.com/5907868/46012034-1bf78a80-c0c8-11e8-835e-d01dd1a6ce51.png)


but you need to layout the aside again so that the buttons are aligned vertically.

Reply:
Thanks for the suggestion 😄! Since that goes beyond pure localization, and it's more of an UX thing, would you mind filing a separate FR and cc @gmajoulet and @newmuis to get this triaged?